conf/defaults.ini

```diff
@@ -505,6 +505,9 @@ global_dashboard = -1
 # global limit of api_keys
 global_api_key = -1
 
+# global limit of api_key seconds to live before expiration
+global_api_key_max_seconds_to_live = -1
```

Comment:
It's a bit weird that this is in the `[quota]` section. This should be moved to either `[security]` or `[auth]` included with updated [configuration documenation](https://github.com/grafana/grafana/blob/master/docs/sources/installation/configuration.md).